docs/resources/deployment.md

```diff
@@ -95,24 +95,23 @@ import {
   to = astro_deployment.imported_deployment
 }
 resource "astro_deployment" "imported_deployment" {
-  original_astro_runtime_version = "11.3.0"
```

Comment:
removed this as it was causing issues with importing deployments. when running `terraform plan` it stated that a new resource was to be created and another to be deleted. removing this field when running tf import fixed it so that it stated a resource was to be updated instead.